README.md

```diff
@@ -38,14 +38,24 @@ Clone the repo and install either via setup.py or wheel (wheel requires less pro
 - `PyYAML==5.4.1`,
 - `yarg==0.1.9`
 
+# GCP Services
+
+AutoMLOps makes use of the following Google Cloud services by default:
+- [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines/introduction)
+- [Artifact Registry](https://cloud.google.com/artifact-registry/docs/overview)
+- [Google Cloud Storage](https://cloud.google.com/storage/docs/introduction)
+- [Cloud Build](https://cloud.google.com/build/docs/overview)
+- [Cloud Build Triggers](https://cloud.google.com/build/docs/triggers)
+- [Cloud Run](https://cloud.google.com/run/docs/overview/what-is-cloud-run)
+- [Cloud Scheduler](https://cloud.google.com/scheduler/docs/overview)
+
 # APIs & IAM
 AutoMLOps will enable the following APIs:
 - [cloudresourcemanager.googleapis.com](https://cloud.google.com/resource-manager/reference/rest)
 - [aiplatform.googleapis.com](https://cloud.google.com/vertex-ai/docs/reference/rest)
 - [artifactregistry.googleapis.com](https://cloud.google.com/artifact-registry/docs/reference/rest)
 - [cloudbuild.googleapis.com](https://cloud.google.com/build/docs/api/reference/rest)
 - [cloudscheduler.googleapis.com](https://cloud.google.com/scheduler/docs/reference/rest)
-- [cloudtasks.googleapis.com](https://cloud.google.com/tasks/docs/reference/rest)
```

Comment:
We are still using Cloud Tasks as of v1.0.1; this needs to be left here until we engineer another solution to submit an authenticated request to Cloud Run